### ISSUE TITLE
Twine Inaccessible For Users Of Screen Readers

### ISSUE THREAD

**Issue by BlindHunter95** [klembot/twinejs#564]
This has been the state of affairs for some time now, and as a result visually impaired persons wishing to use Twine have had to either downgrade to version 1.4.x or use command-line alternatives such as [TweeGo](http://www.motoslave.net/tweego).
Will this ever change? I certainly hope so; blind/visually impaired users should at least have the option to use the graphical twine editor most users know and love.

**Comment by greyelf** [comment]
@BlindHunter95 
To help the developers understand the issues you (and other visually impaired people) are having with the current User Interface or Work Flow, could you please explain in greater detail what those issues are and why they are an issue.

**Comment by BlindHunter95** [comment]
No problem.
Some buttons (such as  Story, Import From File, Archive, etc.) are clearly labeled. Most, however, are not labeled at all (so that when I navigate the interface with the arrow keys or via hotkeys, all my screenreader says is "button, "button, "button.") The good news is that the story list is mostly accessible (accept for the buttons that, according to the source code, switch between light and dark themes.)

**Comment by BlindHunter95** [comment]
Another accessibility issue I forgot to add to my prior comment is this. Most screen readers activate elements in a web browser (and in most graphical UIs) with the bar or enter key. This is not possible when selecting a passage to edit; pressing space or enter does not open the passage.

**Comment by klembot** [comment]
@BlindHunter95, I'm very much interested in making Twine as accessible as I can. I do not have access to screen reader software to test with myself, which makes it difficult to address these issues. Even so, I apologize for the difficulties you and other users have encountered.

In order to make progress on this, as @greyelf points out, we need to track specific issues that can be added to the development roadmap. Your help here in identifying and prioritizing would be greatly appreciated.

I have created a separate issue for screen readers not being able to activate the passage editor button, #565. If you can detail the buttons that don't have labels in a separate issue, as well as other issues, that would be great. I'll tag them a11y (short for accessibility) and begin to add them into the development roadmap.

I'm closing this issue in favor of focusing work on specific problems.

**Comment by Often-Early** [comment]
@klembot The most popular screen readers are bundled with operating systems so you should be able to reproduce this on whatever OS you are running - here is a list https://cccaccessibility.org/assistive-tech/screen-readers
